atom/browser/api/atom_api_web_contents.cc

```diff
@@ -860,7 +860,7 @@ void WebContents::DidGetResourceResponseStart(
       (details.resource_type == content::RESOURCE_TYPE_MAIN_FRAME ||
        details.resource_type == content::RESOURCE_TYPE_SUB_FRAME))
     return;
-  Emit("did-get-response-details",
+  Emit("-did-get-response-details",
```

Comment:
this seems to follow the comments for event deprecation in the method, but instinctively i feel like deprecations should maintain backwards compatibility and this change will break existing listeners 🤔

Reply:
I agree, and that kind of goes against the purpose of the deprecation guard.

Maybe we should emit _both_ events during the interim deprecated phase, then remove the nonprefixed one when we remove the public facing one.

Reply:
The `deprecate.event(...)` additions to `api-web-contents.js` should cause the old event to still fire.

Reply:
oh yep, i see that now! in that case, this looks good to go